addons/mail/static/src/core/common/composer.js

```diff
@@ -159,7 +159,9 @@ export class Composer extends Component {
         );
         useEffect(
             () => {
-                this.ref.el.style.height = this.fakeTextarea.el.scrollHeight + "px";
+                if (this.fakeTextarea.el.scrollHeight) {
+                    this.ref.el.style.height = this.fakeTextarea.el.scrollHeight + "px";
+                }
                 this.saveContentDebounced();
             },
             () => [this.props.composer.text, this.ref.el]
```

Comment:
It could be in master before this PR but the difference only appears in putting comment on a product as you can see [here](https://github.com/odoo/odoo/pull/138233#discussion_r1651169698).